ppa/archive/solr.py

```diff
@@ -91,6 +91,27 @@ class SolrSchema(object):
                 ]
             },
         },
+        {
+            'name': 'punctuation_parser',
+            "class":"solr.TextField",
+            # for now, configuring index and query analyzers the same
+            # if we want synonyms, query must be separate
+            "analyzer": {
+                # "charFilters": [],
+                "tokenizer": {
+                    "class": "solr.WhitespaceTokenizerFactory",
+                },
+                "filters": [
+                    {"class": "solr.StopFilterFactory", "ignoreCase": True,
+                     "words": "lang/stopwords_en.txt"},
+                    {"class": "solr.LowerCaseFilterFactory"},
+                    # {"class": "solr.EnglishPossessiveFilterFactory"},
```

Comment:
Did not include possessive, stemming, or keyword-ing to the punctuation tokenizer. For stemming and keyword-ing, I was afraid "loving," would get stemmed to "love". And possessive would remove apostrophe.